cypress/e2e/tests/migration/migration-waves/crud.test.ts

```diff
@@ -137,6 +137,7 @@ describe(["@tier0"], "Migration Waves CRUD operations", () => {
         // Delete all applications by clicking the delete buttons
         cy.get(applicationTableSelector + " td > button").each((btn) => {
             cy.wrap(btn).click();
+            cy.contains("Delete").click();
```

Comment:
Why didn't you use existing `click()` function?

Reply:
In this case I don't want a 30 secs timeout